presto-main/src/main/java/com/facebook/presto/operator/Driver.java

```diff
@@ -457,7 +457,7 @@ private void destroyIfNecessary()
         }
     }
 
-    private static ListenableFuture<?> isBlocked(Operator operator)
+    private static ListenableFuture<?> getBlockedFuture(Operator operator)
     {
         ListenableFuture<?> blocked = operator.isBlocked();
         if (blocked.isDone()) {
```

Comment:
That is a good point, and a problem with the current code, which I think we should fix now.  I'll add comments to this PR with my suggestions.